quarkus/tests/integration/src/test-providers/resources/com/acme/provider/legacy/jpa/entity/persistence.xml

```diff
@@ -16,18 +16,16 @@
   ~ limitations under the License.
   -->
 
-<persistence version="2.0"
+<persistence version="3.0"
              xmlns="http://java.sun.com/xml/ns/persistence" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-             xsi:schemaLocation="
-        http://java.sun.com/xml/ns/persistence
-        http://java.sun.com/xml/ns/persistence/persistence_2_0.xsd">
+             xsi:schemaLocation="https://jakarta.ee/xml/ns/persistence https://jakarta.ee/xml/ns/persistence/persistence_3_0.xsd">
     <persistence-unit name="user-store" transaction-type="JTA">
         <class>com.acme.provider.legacy.jpa.entity.Realm</class>
         <properties>
             <property name="hibernate.dialect" value="io.quarkus.hibernate.orm.runtime.dialect.QuarkusH2Dialect" />
             <!-- Sets the name of the datasource to be the same as the datasource name in quarkus.properties-->
             <property name="hibernate.connection.datasource" value="user-store" />
-            <property name="javax.persistence.transactionType" value="JTA" />
+            <property name="jakarta.persistence.transactionType" value="JTA" />
```

Comment:
Is this expected? Is it caused by the migration script?

Reply:
@vmuzikar yes, it is caused by the script.